docs/core-concepts.md

```diff
@@ -30,14 +31,36 @@ Here is a sample deployment config for the EC2 operator.
   <img src="./imgs/deployment-config-concept.png" alt="sample deployment config"/>
 </p>
 
-1 :- `api_version` specifies deployment configuration version.
+1:- `api_version` specifies the deployment configuration version.
 
-2 :- `name` Deployment's name. bentoctl recommends to keep deployment name unqiue within each operator to avoid any potential issues.
+2:- `name` Deployment's name. bentoctl recommends keeping the deployment name unique within each operator to avoid potential issues.
 
-3 :- `operator.name` - The name of the operator to used for deployment, see the list of [official operatos](./operator-list.md) supported. If the the operator is not installed it is installed for you based on the version provided. You can also provide a path to the operator if it is in the local directory. This is helpfull if using custom operators.
+3:- `operator.name` - The name of the operator used for deployment, see the list of [official operatos](./operator-list.md) supported. If the operator is not installed it is installed for you based on the version provided. You can also provide a path to the operator if it is in the local directory. This is helpful if using custom operators.
```

Comment:
```suggestion
3:- `operator.name` - The name of the operator used for deployment, see the list of [official operatos](./operator-list.md) supported. If the operator is not already installed, it will be installed automatically based on the version provided. You can also provide a path to the operator if it is in the local directory.
```